chainio/clients/eth/client.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/ethereum/go-ethereum/ethclient"
 )
 
-type gethClient interface {
+type GethClient interface {
```

Comment:
This made me realize, with this new txsender and txmanager etc, is the WaitForTransactionReceipt function below on our EthClient interface really the right place? Or should we expose it on the txsender or txmgr etc? cc @shrimalmadhur

Reply:
Oh yeah txsender exposes the WaitForTransactionReceipt. So let's get rid of the EthClient interface below then?

I'd also vote to name this interface EthClient instead of GethClient, because this works with any EL client, not only geth.

Reply:
done

Reply:
Lgtm but I want to make sure @shrimalmadhur sees this interface breaking change and approves before merging.